index.html

```diff
@@ -606,9 +608,9 @@
 
 	{{#capture}}
 		{{#3}}{{^3.III}}{{^9.I}}{{^7.I}}
-			<p>At game start, all players make a secret bid with their starting money ($0 is a valid bid). Starting with the highest bidder, each player chooses a position in the player order and his capital on the map. In case of a tie, randomly draw lots for the order of choice (the losers get $2 for each lost draw).</p>
+			<p>At game start, all players make a secret bid with their starting money ($0 is a valid bid). {{^9.II}}Starting with the highest bidder, each player chooses a position in the player order and his capital on the map.{{/9.II}}{{#9.II}}{{/9.II}} In case of a tie, randomly draw lots for the order of choice (the losers get $2 for each lost draw).</p>
```

Comment:
Good catch, I was struggling with the rules trying to work out how this combination worked and must have left this for later. I'll fix this now.